### ISSUE TITLE
details about parameters and hyperparameters

### ISSUE THREAD

**Issue by slamandar** [allenai/bilm-tf#21]
Hi, could you share some hyper-parameters details while training biLM？like optimizer, dropout rate? Thanks.

**Comment by matt-peters** [comment]
We used Agagrad with learning rate = 0.2, batch size = 128 * 20 tokens on each GPU across 4 GPUs (with synchronous gradient updates).  Dropout rate was 0.1 on input/output to each LSTM (no recurrent dropout, following Zaremba et al (https://arxiv.org/abs/1409.2329).
